app/Fission/Config/Types.hs

```diff
@@ -12,19 +12,19 @@ import qualified Fission.Storage.Types         as DB
 import qualified Fission.Platform.Heroku.Types as Heroku
 import qualified Network.AWS.Auth              as AWS
 import qualified Fission.AWS.Types              as AWS
-import           Fission.Internal.Orphanage.PGConnectInfo ()
+-- import           Fission.Internal.Orphanage.PGConnectInfo ()
 
 -- | The top level 'Fission' application 'RIO' configuration
-data Config = Config
+data Config dbBackend = Config
```

Comment:
Parameterized this type by the kind of database backend we plug in over in `app`.